consensus/service/src/tx_manager.rs

```diff
@@ -110,11 +113,10 @@ pub trait UntrustedInterfaces: Clone {
     /// * `max_elements` - Maximal number of elements to output.
     ///
     /// Returns a bounded, deterministically-ordered list of transactions that are safe to append to the ledger.
-    fn combine(&self, tx_contexts: &[&WellFormedTxContext], max_elements: usize) -> Vec<TxHash>;
+    fn combine(&self, tx_contexts: &[WellFormedTxContext], max_elements: usize) -> Vec<TxHash>;
```

Comment:
This dodges the need for an explicit lifetime, and turns out to generally simplify things.